lib/src/react16_suggestors/pubspec_react_upgrader.dart

```diff
@@ -50,8 +50,11 @@ class PubspecReactUpdater implements Suggestor {
         if (shouldUpdateVersionRange(
             constraint: constraint, targetConstraint: targetConstraint)) {
           // Wrap the new constraint in quotes if required.
-          var newValue =
-              generateNewVersionRange(constraint, targetConstraint).toString();
+          var newValue = targetConstraint.toString().contains('-alpha') ||
```

Comment:
Can we add a simple test case for this new logic?

Reply:
done